apps/distgo/pkg/script/script.go

```diff
@@ -92,3 +94,20 @@ func WriteAndExecute(buildSpec params.ProductBuildSpec, script string, stdOut, s
 	}
 	return nil
 }
+
+func GetBuildArgs(buildSpec params.ProductBuildSpec, script string) ([]string, error) {
+	stdoutBuf := bytes.Buffer{}
+	stderrBuf := bytes.Buffer{}
+	combinedBuf := bytes.Buffer{}
+	stdoutMW := io.MultiWriter(&stdoutBuf, &combinedBuf)
+	stderrMW := io.MultiWriter(&stderrBuf, &combinedBuf)
```

Comment:
It seems like we only actually use the output of the `stdoutBuf` -- if that's the case, what's the purpose of having the combined buffer or stderr buffer?